acceptance_tests/docker-compose.yaml

```diff
@@ -8,8 +8,22 @@ services:
       MAPSERVER_CATCH_SEGV: 1
     links:
       - db
-    ports:
-      - 8380:8080
+    #ports:
+    #  - 8380:8080
+    volumes:
+      - ./config/mapserver/:/etc/mapserver
+    user: www-data
+
+  mapserver-map:
+    image: camptocamp/mapserver:${DOCKER_TAG}
+    environment:
+      MS_DEBUGLEVEL: 5
+      MAPSERVER_CATCH_SEGV: 1
+      MS_MAPFILE:
```

Comment:
to test if `MS_MAPFILE` is used, you should put the mapfile somewhere else and set this variable to the mapfile

Reply:
? I want to test the MAP argument...